docs/i18n/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+title: i18n
+permalink: /i18n/
+has_children: true
+nav_order: 1
+---
+
+i18n
```

Comment:
probably add some content on that initial page so it doesn't look so "empty"?

Reply:
should I better take "Overview" as nav_order:1 for now?

Reply:
sure - but then the chapter has only one sub-page.
it's considered not-so-good practice to have a chapter (list) w/ only one page (list item).
so try to think of a layout that either has
- 2 subpages with an overview text/page (yeah!)
- single page with all content (meh)

Reply:
it´s just the beginning, so i plannend more subpages, that´s why i created this subpages

Reply:
👍